Sources/SwiftProtobuf/Message+JSONAdditions.swift

```diff
@@ -93,3 +93,72 @@ public extension Message {
   }
 }
 
+
+public extension Array where Iterator.Element: Message {
```

Comment:
Yet another option:  We could support this by providing a facility to initialize message from `JSONSerialization`-decoded data.  That would provide a reusable building block for folks decoding all sorts of JSON structures.